Client/Frontend/Reader/ReaderModeHandlers.swift

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct ReaderModeHandlers {
+    static func register(webServer: WebServer) {
+        // Register our fonts, which we want to expose to web content that we present in the WebView
+        webServer.registerMainBundleResourcesOfType("ttf", module: "reader-mode/fonts")
+
+        // Register the handler that accepts /reader-mode/page?url=http://www.example.com requests
+        webServer.registerHandlerForMethod("GET", module: "reader-mode", resource: "page") { (request: GCDWebServerRequest!) -> GCDWebServerResponse! in
```

Comment:
So we're using localhost as an HTTP URL, which I think should be generally safe in terms of XHRs since we don't host any other content at localhost. Of course, once we add other handlers for serving local data, this could become dangerous again.

Doing a bit of research on the topic, I found the [Content-Security-Policy](http://www.ibuildings.com/blog/2013/03/4-http-security-headers-you-should-always-be-using) HTTP header that could be quite useful here. If we can attach the proper header to the data response, I think that should eliminate any XSS security threats. I'll mention this in [bug 1143180](https://bugzilla.mozilla.org/show_bug.cgi?id=1143180) too.